designsafe/static/scripts/projects/components/manage-experiments/manage-experiments.component.js

```diff
@@ -333,39 +337,26 @@ class ManageExperimentsCtrl {
     saveExperiment($event) {
         $event.preventDefault();
         this.data.busy = true;
-        // This modal needs to be refactored.
+        // This modal needs some tidying up.
         // we will never be adding more than one experiment at a time.
-        // there is also no need for entity management within this modal.
         this.form.addExperiments[0].authors = this.data.users;
-        if (this.form.addExperiments[0].procedureStart && !this.form.addExperiments[0].procedureEnd) {
-            this.form.addExperiments[0].procedureEnd = this.form.addExperiments[0].procedureStart;
-        }
-        let addActions = this.form.addExperiments.filter((exp) => (exp.title && exp.experimentalFacility && exp.experimentType)).map((exp) => {
-            exp.description = exp.description || '';
-            return this.ProjectEntitiesService.create({
-                data: {
-                    uuid: this.data.project.uuid,
-                    name: 'designsafe.project.experiment',
-                    entity: exp,
-                },
-            }).then((res) => {
-                this.data.project.addEntity(res);
-                this.data.experiments = this.data.project.experiment_set;
-                this.data.users.forEach((user) => {
-                    user.authorship = false;
-                });
+        var experiment = this.form.addExperiments[0];
```

Comment:
Should this be declared as `let`?
Line 95 in `designsafe/static/scripts/projects/components/manage-simulations/manage-simulations.component.js` also uses `var`.

Reply:
Oh yep, I'll swap that out

Reply:
Should be sorted out now among other things.